tests/testthat/model-examples/1/Run_1.o1

```diff
@@ -1,9 +0,0 @@
-time="2020-05-20T16:45:27-04:00" level=info msg="Successfully loaded default configuration from /data/home/sethg/rbabylon/tests/testthat/model-examples/1/babylon.yaml"
```

Comment:
This is fine. This file is created when the model is run on the grid, which apparently it was last time and wasn't this time. Doesn't matter for our tests.